### PR TITLE
CP-328 Create src,test dirs on gulp-init, add detail to README

### PR DESCRIPTION
**Pull request by robbecker-wf**
I added to the gulp-init script so that it creates the src and test directories when run. If you follow the instructions to create a project from scratch and then run gulp, it immediately fails because there is no src directory. 

We could consider moving some of the generator functionality into the gulp-init script.

To test: 
- Checkout this PR branch.
- Create a new directory and cd into it
```
npm init
npm install workiva/wGulp --save-dev
./node_modules/.bin/gulp-init
``` 
It should create the test and src directory.

### CODE REVIEW

**Comment by rmconsole-wf** [comment]
When this pull is merged I will not be able to function
		properly because no ticket is included in the title.

***
*Last updated on Monday, January 12 03:30 PM CST*

File: README.md [hunk 1]
```diff
@@ -77,6 +74,9 @@ The best and quickest way to get value out of wGulp is to adhere to a standard d
 * `./dist` - Destination for generated distribution assets
 * `./docs` - Destination for generated documentation
 * `./report` - Destination for generated reports (coverage, complexity, test)
+* `./sass` - Default location for OOCSS styles
+* `./src` - TypeScript, JavaScript, JSX, Coffee, ... source code
```
Comment (robbecker-wf) [inline README.md]: Alphabetized and added more description to the src directory to make it clear that all other languages that need to be transpiled also live in src.

**Comment by trentgrover-wf** [comment]
@maxwellpeterson-wf @evanweible-wf

**Comment by robbecker-wf** [comment]
CI failing through no change of mine. :(

**Comment by tomconnell-wf** [comment]
That CI failure is wack.  How does that relate to this change?  Looks like it is failing on master.

+1  
ermahgerd-clarinet

**Comment by trentgrover-wf** [comment]
+1 - fixed the broken test fixture (just from differences in React 0.12.2)

@jayudey-wf @charliekump-wf ready for merge and minor patch release

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by evanweible-wf** [comment]
+1

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - verified that when running init a test and src directory are created with update
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into master.

**Comment by tomconnell-wf** [comment]
+10

I differed from the instructions a bit, as this branch isn't in the npm registry.  I just npm installed a local directory.

 mkdir test
 cd test
 npm init
 cd ..
 cd wGulp
 git clone git@github.com:Workiva/wGulp.git
 cd wGulp
 gco more-init
 cd ..
 cd test
 npm install ../wGulp/
 ./node_modules/.bin/gulp-init

src/ and test/ are there.

**Comment by charliekump-wf** [comment]
@rmconsole-wf

**Comment by charliekump-wf** [comment]
Merging.

**Pull request merged**